tests/system/apps/__init__.py

```diff
@@ -15,24 +15,39 @@ def load_app(app_name):
     return app
 
 
-def mesos_app():
-    return load_app('mesos-app')
+def mesos_app(app_id=None):
+    if app_id is None:
```

Comment:
This app ID generation is already done in the same way by `load_app` which in turn calls `make_id`. Should it be like
```
if app_id_prefix is None: app_id_prefix = ...
return load_app(app_id_prefix)
```
?

Reply:
The app name is also the name of the JSON file but you are right. I'll refactor this.